app/views/assessor_interface/assessment_sections/show.html.erb

```diff
@@ -46,10 +46,19 @@
   <% end %>
 <% end %>
 
-<% if @assessment_section_view_object.show_online_checker? %>
-  <h2 class="govuk-heading-s">Online checker</h2>
-  <p class="govuk-body">This authority has an online checker for validating the supplied reference number:</p>
-  <p class="govuk-body"><%= govuk_link_to @assessment_section_view_object.online_checker_url, @assessment_section_view_object.online_checker_url, target: "_blank" %></p>
+<% if @assessment_section_view_object.professional_standing? %>
+  <%= govuk_accordion do |accordion| %>
+    <% if @assessment_section_view_object.show_online_checker? %>
+      <% accordion.section(heading_text: "Online checker") do %>
+        <p class="govuk-body">This authority has an online checker for validating the supplied reference number:</p>
+        <p class="govuk-body"><%= govuk_link_to @assessment_section_view_object.online_checker_url, @assessment_section_view_object.online_checker_url, target: "_blank" %></p>
+      <% end %>
+    <% end %>
+
+    <% accordion.section(heading_text: "Competent authority information shown to applicant") do %>
+      <%= render "shared/eligible_region_content_components/professional_recognition", region: @assessment_section_view_object.application_form.region %>
```

Comment:
```suggestion
      <%= render "shared/eligible_region_content_components/professional_recognition", region: @assessment_section_view_object.region %>
```

This is available on the view object directly:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/ded278a4baace9d0e94a1ed0a6ca051b9a4f86d2/app/view_objects/assessor_interface/assessment_section_view_object.rb#L26